newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/ExamplesLine.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react';
+import { type ExampleShortHeader } from '../../../../Utils/GDevelopServices/Example';
+import { Column } from '../../../../UI/Grid';
+import Text from '../../../../UI/Text';
+import { LineStackLayout } from '../../../../UI/Layout';
+import FlatButton from '../../../../UI/FlatButton';
+import { Trans } from '@lingui/macro';
+import Add from '../../../../UI/CustomSvgIcons/Add';
+import ExamplesGrid from './ExamplesGrid';
+import { prepareExamples } from '../../../../AssetStore/ExampleStore';
+
+type ExamplesLineProps = {|
+  exampleShortHeaders: Array<ExampleShortHeader>,
+  onExpand: () => void,
```

Comment:
Ubert nit: onShowAll (to align with the name of the button)?

Reply:
Yeah I thought that if we change the wording to "see all" one day, we'd be happy the code is not linked to the wording.
But I think showAll describes well the action too